website/content/docs/k8s/multiport/traffic-split.mdx

```diff
@@ -34,13 +34,7 @@ Complete the following steps to implement a split in TCP traffic between two ser
 
 ## Define route resource
 
-The following example splits traffic for the services in the `api` Pod.
-
-<Tabs>
-
-<Tab heading="Method 1" group="method1">
-
-TCP traffic for services registered to the Consul catalog that are available at the `admin` port is split so that 50% of the traffic routes to the service at the `api` port and 50% routes to the service at the `admin` port.
+The following example splits traffic for the `api` service. TCP traffic for services registered to the Consul catalog that are available at the `api-workload` port is split so that 50% of the traffic routes to the service at the `api-workload` port and 50% routes to the service at the `admin-workload` port.
```

Comment:
This is essentially the same example as in `tcproute.mdx`. The difference between the two "methods" was just port names (they're not functionally different), so I'm not clear on why we have both... I think the correct adjustment is to align w/ the example in `tcproute.mdx` and drop the second variant, so I've done that.